.commitlintrc.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Will this rule only apply to new commit ?

Reply:
Unfortunately yes :(

Reply:
Fortunately yes lol. Cause we have some old commit doesn't follow the conventional commits rules ...